src/main/java/org/sagebionetworks/bridge/sdk/BridgeAdminClient.java

```diff
@@ -33,6 +33,19 @@ public boolean createUser(SignUpByAdmin signUp) {
     }
 
     @Override
+    public boolean signOutUser(String email) {
+        session.checkSignedIn();
+        checkArgument(isNotBlank(email));
+
+        Map<String, String> queryParams = new HashMap<String, String>();
+        queryParams.put("email", email);
+
+        HttpResponse response = post(config.getUsersSignOutApi() + toQueryString(queryParams));
+
+        return response.getStatusLine().getStatusCode() == 200;
```

Comment:
I don't believe this is necessary. If the request does not return 200, an exception will be thrown. So unless we're returning data, we can return void.